plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -120,22 +128,50 @@ export function WriteItForMeButtonContainer() {
 	const writeItForMeEnabled =
 		! fetching && productTitle.length >= MIN_TITLE_LENGTH_FOR_DESCRIPTION;
 
-	const buildPrompt = () => {
-		const instructions = [
-			`Write a product description with the following product title: "${ productTitle.slice(
+	const buildPrompt = (): string => {
+		const productName: string = getProductName();
+		const productCategories: string[] = getCategories();
+		const productTags: string[] = getTags();
+		const productAttributes: Attribute[] = getAttributes();
+
+		const includedProps: string[] = [];
+		const productPropsInstructions: string[] = [];
+		if ( productCategories.length > 0 ) {
+			productPropsInstructions.push(
+				`Falling into the categories: ${ productCategories.join(
+					', '
+				) }.`
+			);
+			includedProps.push( 'categories' );
+		}
+		if ( productTags.length > 0 ) {
+			productPropsInstructions.push(
+				`Tagged with: ${ productTags.join( ', ' ) }.`
+			);
+			includedProps.push( 'categories' );
+		}
+		productAttributes.forEach( ( { name, values } ) => {
+			productPropsInstructions.push(
```

Comment:
I'm noticing that for attributes, the prompt doesn't say that they're attributes, but just prints them out to the screen. Do you think we should mention what these are, like you do here with other types of data?

Reply:
My thinking is that the attribute name should be enough to describe what the attribute is. I don't think adding the word "Attribute" would help much. For instance, saying "Color: Red" is clearer and closer to natural language IMO than saying "Attributes: Name: Color, Value: Red".